Package.swift

```diff
@@ -96,6 +96,11 @@ let package = Package(
       url: "https://github.com/apple/swift-protobuf.git",
       from: "1.14.0"
     ),
+    .package(
+      name: "GoogleAppMeasurement",
+      url: "https://github.com/google/GoogleAppMeasurement.git",
+      from: "7.4.0"
```

Comment:
Do we want to do exact versioning here, or are they going to release in a different cadence?

Reply:
There is still a requirement to do exact matching, but the Analytics team will explore loosening in the future. cc: @allenkv @htcgh

Reply:
In that case we should use `.package(name: "GoogleAppMeasurement", url: "https://github.com/google/GoogleAppMeasurement.git", .exact("7.4.0")` (if that's the right syntax)

Reply:
Good catch. Thanks! Done.